shared/app/push-prompt.native.tsx

```diff
@@ -13,73 +15,110 @@ type Props = {
-        src={require('../images/illustrations/illustration-turn-on-notifications-460-x-252.png')}
-      />
-      <Kb.Text center={true} type="BodySmallSemibold" style={styles.text}>
-        It's{' '}
-        <Kb.Text center={true} type="BodySmallSemiboldItalic" style={styles.text}>
-          very
-        </Kb.Text>{' '}
-        important you enable notifications.
-      </Kb.Text>
-      <Kb.Text center={true} type="BodySmall" style={styles.text}>
-        This phone may need to perform crypto for you, which the Keybase servers cannot do. For example, if
-        you provision a new device, this phone will be contacted.
-      </Kb.Text>
+  <Kb.Box2 direction="vertical" fullHeight={true} fullWidth={true} gap="small" style={styles.container}>
+    <Kb.RequireImage
+      resizeMode="stretch"
+      style={styles.image}
+      src={require('../images/illustrations/illustration-turn-on-notifications-460-x-252.png')}
+    />
+    <Kb.Text center={true} type="BodySmallSemibold" style={styles.text}>
+      It's{' '}
+      <Kb.Text center={true} type="BodySmallSemiboldItalic" style={styles.text}>
+        very
+      </Kb.Text>{' '}
+      important you enable notifications.
+    </Kb.Text>
+    <Kb.Text center={true} type="BodySmall" style={styles.text}>
+      This phone may need to perform crypto for you, which the Keybase servers cannot do. For example, if you
+      provision a new device, this phone will be notified.
+    </Kb.Text>
+    <Kb.Box2 direction="vertical" fullWidth={true} style={styles.buttonContainer}>
       <Kb.WaitingButton
         fullWidth={true}
         onClick={props.onRequestPermissions}
-        label="Got it"
+        label="Allow notifications"
         waitingKey={Constants.permissionsRequestingWaitingKey}
         style={styles.button}
+        type="Success"
       />
-      <Kb.Button
-        type="Dim"
-        fullWidth={true}
-        onClick={props.onNoPermissions}
-        label="No thanks"
-        style={styles.button}
-      />
-      <Kb.Box style={styles.spacer} />
     </Kb.Box2>
-  </Kb.ScrollView>
+  </Kb.Box2>
 )
 
-const styles = Styles.styleSheetCreate({
-  button: {maxHeight: 40},
-  container: {padding: Styles.globalMargins.small},
-  image: Styles.platformStyles({
-    isMobile: {
-      height: 200,
-      width: '170%',
-    },
-  }),
-  scroll: {
-    backgroundColor: Styles.globalColors.white,
-    ...Styles.globalStyles.fillAbsolute,
-  },
-  scrollContent: {minHeight: '100%'},
-  spacer: {flexGrow: 1},
-  text: {
-    color: Styles.globalColors.black,
-  },
-})
-
 const mapStateToProps = () => ({})
 const mapDispatchToProps = dispatch => ({
   onNoPermissions: () => dispatch(PushGen.createRejectPermissions()),
   onRequestPermissions: () => dispatch(PushGen.createRequestPermissions()),
 })
 
+const PushPromptWithHeader = Container.withProps(
+  (props: Props & HeaderHocProps & ReturnType<typeof mapDispatchToProps>): Partial<HeaderHocProps> => ({
+    customComponent: (
```

Comment:
This title isn't appearing
<img width="350" alt="image" src="https://user-images.githubusercontent.com/11968340/63354612-be4cdc00-c332-11e9-9dca-2d5dd90bf751.png">
Also, I think you could do this using `titleComponent`, `headerStyle` etc. rather than using `customComponent`.

Reply:
I tried using `titleComponent`, but it wouldn't center.

Reply:
Yeah I have to leave it as `customComponent`, at least until the refactor.

Reply:
@buoyad did you try using storybook or the signup flow? I see the title in storybook...

Reply:
The storybook does not accurately represent the screen. It can be different in the app

Reply:
OK I tested in-app and it shows me the title:
![36c2e9c2-d366-4a18-af5c-b3d684cbf361](https://user-images.githubusercontent.com/59594/63392846-33d59e00-c36d-11e9-8a9b-e9d67b3d5cd4.png)